src/md5.js

```diff
@@ -192,16 +192,16 @@ const md5 = class MD5 {
          * @returns {string}
          */
         function calculate(data) {
-            let data_length = data.length
-            let state = [1732584193, -271733879, -1732584194, 271733878]
+            const data_length = data.length
+            const state = [1732584193, -271733879, -1732584194, 271733878]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The numeric literal '271733879' will have at different value at runtime.](https://app.codacy.com/manual/joaocac/impulses/pullRequest?prid=4987369)